src/browser.js

```diff
@@ -148,35 +204,20 @@ function save(namespaces) {
  */
 
 function load() {
-  try {
-    return exports.storage.debug;
-  } catch(e) {}
-
-  // If debug isn't set in LS, and we're in Electron, try to load $DEBUG
-  if (typeof process !== 'undefined' && 'env' in process) {
-    return process.env.DEBUG;
+  if (exports.storage) {
+    exports.storage.get('debug', exports.enable);
```

Comment:
Another problem I am seeing is that `enabled` is running **before** `enable` runs here, and thus `exports.names` are not set when it is checking for what to enable. This breaks the `enabled` check and ensures logging is disabled for the duration of the page load.